.idea/gradle.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would say we should drop the entire `.idea` folder - it's not necessary for the quickstarts.

Reply:
I agree - removed it!